ccd-definition/AuthorisationCaseField/AuthorisationCaseField.json

```diff
@@ -1784,7 +1784,8 @@
     "AccessControl": [
       {
         "UserRoles": [
-          "[RESPONDENTSOLICITORONE]"
+          "[RESPONDENTSOLICITORONE]",
+          "[RESPONDENTSOLICITORONESPEC]"
```

Comment:
The AP will get merged today needing some rework here as we will not be using the traditional CaseRoles
https://github.com/hmcts/civil-ccd-definition/pull/1267/files